doc/FREQUENTLY_MADE_SUGGESTIONS.md

```diff
@@ -106,6 +106,18 @@ In the end, it's just odd to be able to hold 10000 rounds in your hands at once.
 
 Again, we do not want to make the game worse with this - there are some unavoidable problems caused by this migration, but after it is done, not only it would be easier to contribute for everyone, but the game would work better, and maybe even faster.
 
+### Options
+
+#### We should be able to disable features we don't want (portal storms, etc.): No.
+
+Simply put, features make it into the game because they are desired features which contribute to the game's intended direction.
+
+Features are shipped in stable versions when they are deemed to be in a good enough state. Features in experimental may come with bugs, severe balance issues, so on. That is the nature of experimental.
```

Comment:
I don't think this reflects reality?
The wording implies that features get *ported* from experimental to stable, when in fact stable branch gets created at an arbitrary point in time, and gets polished from there. Which might lead to the same eventual outcome ("features in stable are in good state"), but that's still different.

Reply:
There are selective reverts for stable or other patching mechanisms if we're not happy with how the feature works. See for example #74492

While this is not *technically* the same thing as "It will not be put onto the stable branch" it is effectively the same thing, in that the reader (the stable player) will only experience the features if we think they're in a good enough state.

Reply:
I stand by the fact that the current wording is disingeneous and misleading.

You don't need to explain how git works - you can put it in lay terms. Something like "A stable version is only released/is only declared as "stable" once we are satisfied with the state of all the features within"